### PR TITLE
Ensure config file exists before loading configuration in imagePolicyWebhook

### LINKED ISSUE #55378: Apiserver would panic if ImagePolicyWebhook enabled and no configuration provided
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug

**What happened**:
If start apiserver with ImagePolicyWebhook  admission enabled , and no configuration provided, apiserver would panic:
```
I1109 15:30:58.799478  104950 initialization.go:90] enabled Initializers feature as part of admission plugin setup
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x54ec0a]

goroutine 1 [running]:
bufio.(*Reader).fill(0xc420b75380)
	/usr/local/go/src/bufio/bufio.go:97 +0xfa
bufio.(*Reader).Peek(0xc420b75380, 0x1000, 0x1000, 0xc42118e000, 0x1000, 0x1000, 0xc420f9e8e8)
	/usr/local/go/src/bufio/bufio.go:129 +0x67
k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/yaml.GuessJSONStream(0x0, 0x0, 0x1000, 0xc420022000, 0xc420f9e9c8, 0x4823d4, 0x47f295, 0x1, 0x2)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/yaml/decoder.go:329 +0x8a
k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/yaml.(*YAMLOrJSONDecoder).Decode(0xc420f9ec08, 0x3bcf160, 0xc4211717a0, 0xa17d4f8, 0xc42106d3f0)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/yaml/decoder.go:218 +0x4e9
k8s.io/kubernetes/plugin/pkg/admission/imagepolicy.NewImagePolicyWebhook(0x0, 0x0, 0xc420f9ec98, 0xc4206b21b0, 0xc42001f340)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/plugin/pkg/admission/imagepolicy/admission.go:225 +0xbb
k8s.io/kubernetes/plugin/pkg/admission/imagepolicy.Register.func1(0x0, 0x0, 0x0, 0x0, 0x1, 0x0)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/plugin/pkg/admission/imagepolicy/admission.go:53 +0x39
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/admission.(*Plugins).getPlugin(0xc420167030, 0xc420f4feac, 0x12, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/admission/plugins.go:103 +0x182
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/admission.(*Plugins).InitPlugin(0xc420167030, 0xc420f4feac, 0x12, 0x0, 0x0, 0x9e0d6a0, 0xc42117eca0, 0x1, 0xc42117ecc0, 0x0, ...)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/admission/plugins.go:149 +0x7d
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/admission.(*Plugins).NewFromPlugins(0xc420167030, 0xc420874900, 0x9, 0x9, 0x9e0d720, 0xc420b85180, 0x9e0d6a0, 0xc42117eca0, 0x0, 0x411a42, ...)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/admission/plugins.go:131 +0x126
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/options.(*AdmissionOptions).ApplyTo(0xc420acec60, 0xc420f48d00, 0x9e56f80, 0xc42108f830, 0xc4203fd4a0, 0xc42001fec0, 0xc42117ec60, 0x2, 0x2, 0xc42116f0e0, ...)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/options/admission.go:110 +0x4b8
k8s.io/kubernetes/cmd/kube-apiserver/app.BuildGenericConfig(0xc42040a000, 0xc420a7ec30, 0x0, 0x0, 0x2, 0x1, 0x0, 0x1, 0xc420a3c410, 0xc420fcef30, ...)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/kube-apiserver/app/server.go:480 +0x106c
k8s.io/kubernetes/cmd/kube-apiserver/app.CreateKubeAPIServerConfig(0xc42040a000, 0x0, 0x0, 0xc420a7ec30, 0x0, 0x0, 0xc420274a80, 0x5f, 0xb1, 0x5f, ...)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/kube-apiserver/app/server.go:268 +0xb0
k8s.io/kubernetes/cmd/kube-apiserver/app.CreateServerChain(0xc42040a000, 0xc420acf2c0, 0xc420f9f908, 0x1, 0x1)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/kube-apiserver/app/server.go:133 +0x8d
k8s.io/kubernetes/cmd/kube-apiserver/app.Run(0xc42040a000, 0xc420acf2c0, 0xc420f9f9e8, 0xc420fcee10)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/kube-apiserver/app/server.go:119 +0xfa
main.NewKubeAPIServer.func1(0xc420acf500, 0xc420fad800, 0x0, 0x20, 0xc420acf2c0, 0x0, 0xc420acf380)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/hyperkube/kube-apiserver.go:35 +0x34
main.(*HyperKube).Run(0xc420988480, 0xc420010250, 0x20, 0x20, 0xc420acf2c0, 0x0, 0x0)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/hyperkube/hyperkube.go:193 +0x408
main.(*HyperKube).RunToExit(0xc420988480, 0xc420010240, 0x22, 0x24)
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/hyperkube/hyperkube.go:205 +0x57
main.main()
	/workspace/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/hyperkube/main.go:49 +0x5ec
```

**What you expected to happen**:
Return error in an elegant way

/sig apps

### PR DESCRIPTION
**Pull request by lichuqiang**
**What this PR does / why we need it**:
ensure config file exists before loading configuration in imagePolicyWebhook

**Which issue(s) this PR fixes**
Fixes #55378

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: plugin/pkg/admission/imagepolicy/admission.go [hunk 1]
```diff
@@ -219,6 +219,10 @@ func (a *Plugin) admitPod(pod *api.Pod, attributes admission.Attributes, review
 // For additional HTTP configuration, refer to the kubeconfig documentation
 // http://kubernetes.io/v1.1/docs/user-guide/kubeconfig-file.html.
 func NewImagePolicyWebhook(configFile io.Reader) (*Plugin, error) {
+	if configFile == nil {
+		return nil, fmt.Errorf("Config file empty")
```
Comment (deads2k) [inline plugin/pkg/admission/imagepolicy/admission.go]: nit: "no config specified"

**Comment by deads2k** [comment]
nit, then lgtm

/approve no-issue

**Comment by lichuqiang** [comment]
@deads2k Done, thanks. 
So, could you give it a LGTM?

File: plugin/pkg/admission/imagepolicy/admission.go [hunk 2]
```diff
@@ -219,6 +219,10 @@ func (a *Plugin) admitPod(pod *api.Pod, attributes admission.Attributes, review
 // For additional HTTP configuration, refer to the kubeconfig documentation
 // http://kubernetes.io/v1.1/docs/user-guide/kubeconfig-file.html.
 func NewImagePolicyWebhook(configFile io.Reader) (*Plugin, error) {
+	if configFile == nil {
+		return nil, fmt.Errorf("No config specified")
```
Comment (deads2k) [inline plugin/pkg/admission/imagepolicy/admission.go]: sorry, the case in my message https://github.com/kubernetes/kubernetes/pull/55379#discussion_r151248793 was important.  `fmt.Errorf` message always start lower case and end without punctuation.

Comment (lichuqiang) [reply plugin/pkg/admission/imagepolicy/admission.go]: Oh, gotcha. I might have seen lots of wrong demonstration :)

**Comment by lichuqiang** [comment]
@deads2k again done :)

**Pull request merged**